### ISSUE TITLE
Highlighting the active page in a navbar

### ISSUE THREAD

**Issue by retorquere** [jgthms/bulma#1638]
This is about **Bulma**.

This is a question/I need help.

I have a navbar, and I want to highlight the active menu entry. I can set extra classes on the active menu (I currently add `is-selected` but I could add any other); I'd want to achieve the visual effects of `is-active` but without the menu remaining dropped-down. Is this possible?

This is about the Bulma **CSS framework**
I'm using Bulma **version** 0.6.2
My **browser** is: Chrome
I am (fairly) sure this issue is **not a duplicate**

**Comment by eppfel** [comment]
Are you talking about the dropdown menu or the burger-menu? Because with the latter using `is-active` on `navbar-item` is no problem.

**Comment by retorquere** [comment]
About the dropdown menu, sorry about omitting that info.

**Comment by prodigeris** [comment]
Also interested in this.

**Comment by rodrigoaguilera** [comment]
This might be a good chance to make the `a.navbar-item.is-active` style more noticeable on the default bulma skin (no coloring of the navbar) since it goes from #0a0a0a (active) to #4a4a4a (not active).
Also the active background is transparent. Should it be some kind of grey?

I hope I explained myself clearly. I can provide some HTML samples if necessary.

**Comment by Lucretiel** [comment]
Can I request that .navbar-item.is-active gets an example in the documentation?

**Comment by suud** [comment]
@Lucretiel what kind of example are you missing?

The last paragraph of the [navbar-item](https://bulma.io/documentation/components/navbar/#navbar-item) section in the documentation already mentions:

> You can add the following modifier classes:
> - `is-expanded` to turn it into a full-width element
> - __`is-tab` to add a bottom border on hover and show the bottom border using `is-active`__

Adding `is-tab` to my `a.navbar-item`'s fixed the visibility problem mentioned by @rodrigoaguilera, for me. The item having `is-active` is noticeably highlighted, now.

**Comment by alexanderadam** [comment]
> since it goes from #0a0a0a (active) to #4a4a4a (not active)

Indeed. It seems that the current rule is simply to [darken the color by 5%](https://github.com/jgthms/bulma/blob/5198384d0260eeadd69e6c416bcca8f5e394e6ab/sass/components/navbar.sass#L75).
Accessibility wise [this clearly fails](https://webaim.org/resources/contrastchecker/?fcolor=4A4A4A&bcolor=0A0A0A). Especially since this is only text and not an area.

Since the 5%-logic clearly fails, you need to add another hint for activation (i.e. [migrating to a tab look as mentioned before](https://github.com/jgthms/bulma/issues/1638#issuecomment-652904624)) or you can visually 'inactivate' the inactive colour even more. For instance something like this:

```css
.navbar-menu .navbar-item, .navbar-link {
  color: #9C9C9C;
}
```

![Screenshot from 2020-08-20 17-11-47](https://user-images.githubusercontent.com/372620/90790375-467c0d80-e308-11ea-8387-42ef80052ff5.png)

**Comment by Aldo-f** [comment]
Or something like this: (same styling as an item when hovering)
```css
nav.navbar a.active {
    background-color: hsl(0deg, 0%, 98%);
    color: #FA7C91;
}
```

![image](https://user-images.githubusercontent.com/5683799/181592741-b5cd5aef-1cf4-4bba-b93b-5af5609b1f43.png)


See example:
[CodePen - active navbar-items - Bulma](https://codepen.io/AldoF/pen/bGvYBzo) 


PS: class `is-active` is used for opening a hoverable item, not for showing if the current nav-item is the active page.

**Comment by richardtorres314** [comment]
> You can add the following modifier classes:
> * is-expanded to turn it into a full-width element
> * is-tab to add a bottom border on hover and show the bottom border using is-active

Am I missing something here when I read this? There is definitely no border being applied to `a.navbar-item.is-active`. If the intent is for `is-active` to be applied as a modifier _only_ for dropdowns, the point stated above in the docs is incorrect.

**Comment by shriharip** [comment]
> > You can add the following modifier classes:
> > 
> > * is-expanded to turn it into a full-width element
> > * is-tab to add a bottom border on hover and show the bottom border using is-active
> 
> Am I missing something here when I read this? There is definitely no border being applied to `a.navbar-item.is-active`. If the intent is for `is-active` to be applied as a modifier _only_ for dropdowns, the point stated above in the docs is incorrect.

I agree and did check this. I think docs are wrong. We need to have both `is-active` and  `is-tab` to have bottom border.
